readalongs/align.py

```diff
@@ -53,7 +53,7 @@
 
 @dataclass
 class WordSequence:
-    """ Sequence of "unit" XML elements
+    """Sequence of "unit" XML elements
```

Comment:
You're the second person to remove my space between `"""` and the text. I assume your editor did it automatically? When I run black, it doesn't change this, but I'll try to remember no to insert that space in the future.

Reply:
Hm. interesting. Yes it was done automatically but I assumed it was black. Maybe it's vscode. I'll pay more attention next time.